src/TsTypes.ts

```diff
@@ -1,5 +1,9 @@
 import { camelCase, upperFirst } from 'lodash'
 
+const multiLineCommentStart =  '/** '
+const multiLineCommentIndent = '    '
```

Comment:
BTW: If you like multi line comments like this:
```
/**
  *
  *
*/
```

its a simple change... Just define: `const multiLineCommentIndent = '  * '`